deploy/rhsm-clowdapp.yaml

```diff
@@ -204,6 +204,12 @@ parameters:
     value: 'false'
   - name: RH_MARKETPLACE_BILLABLE_USAGE_KAFKA_SEEK_OVERRIDE_TIMESTAMP
     value: ''
+  - name: SPRING_ACTIVEMQ_BROKER_URL
+    value: vm://localhost?broker.persistent=false
```

Comment:
Looks like these might need to be updated before merging.

Reply:
This provides the default, and given lack of connectivity from ephemeral environments, this is the only sane default at the moment :-). I did go ahead and set the proper values in app-interface.